README.md

```diff
@@ -14,7 +14,7 @@ Prerequisites: JDK 11, update IntelliJ to the most recent version.
    1. Select the project directory, and click `OK`
    1. If there are any further prompts, accept the defaults.
 1. Configure the project to use **JDK 11** (not other versions) as explained in [here](https://www.jetbrains.com/help/idea/sdk.html#set-up-jdk).
-1. After that, locate the `src/main/java/Duke.java` file, right-click it, and choose `Run Duke.main()`. If the setup is correct, you should see something like the output below.
+1. After that, locate the `src/main/java/duke.java` file, right-click it, and choose `Run duke.main()`. If the setup is correct, you should see something like the output below.
```

Comment:
Perhaps you could distinguish between class names and non-class names:  [https://se-education.org/guides/conventions/java/basic.html#naming](https://se-education.org/guides/conventions/java/basic.html#naming)